docs/list.md

```diff
@@ -193,6 +193,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: All the [Hacktoberfest tagged issues](https://github.com/DagsHub/Hacktoberfest-Issues).
 - **Notes**: Other issues and repos of various difficulty listed in the [announcement post](https://dagshub.com/blog/hacktoberfest-2023/).
 
+#### **Depths**
+
+- **Swag**: Depth's T-shirt,Cap,Sticker and Notebook
+- **Requirements**:
+   - Complete the "#30DaysofOpenSource" Challenge by reading daily and giving insights into it.
+   - Have at least one pull request merged in October as part of the Hacktoberfest challenge.
```

Comment:
Hey, I am sorry I think I should have added more indentations for it to render properly as the rest of the list. :)
If you check the deploy preview from the netlify bot, you can see how it will look when it goes live.

I have now added more spaces to it. If this suggestion still doesn't fix it, please indent it as required. Each time this PR is updated, a new deploy preview is generated. So you can check the new preview.

Thank you.

```suggestion
    - Complete the "#30DaysofOpenSource" Challenge by reading daily and giving insights into it.
    - Have at least one pull request merged in October as part of the Hacktoberfest challenge.
```